queries.sql

```diff
@@ -0,0 +1,54 @@
+-- queries for the given questions
+--Question 1:
+SELECT sender_id, receiver_id, amount, created_at 
+FROM Transactions
+WHERE created_at <= now() AND created_at >= given_date;
+
+SELECT account_id, amount, balance, created_at 
+FROM Withdraws/Deposits
+WHERE created_at <= now() AND created_at >= given_date;
+
+--Question 2
+--- for converting into json format
+SELECT sender_id, receiver_id, amount, created_at 
```

Comment:
Need to include customer, account and transaction information. This of it like bank statement.

Reply:
account information and transaction information are already included for customer information I will have to join accounts and customer tables also.